common/app/layout/Front.scala

```diff
@@ -170,7 +173,8 @@ object FaciaContainer {
     config: CollectionConfigWithId,
     collectionEssentials: CollectionEssentials,
     containerLayout: Option[ContainerLayout],
-    componentId: Option[String]
+    componentId: Option[String],
+    omitMPU: Boolean = false
```

Comment:
I was afraid of making too many changes around this code, but I think you're right.  I'll change it.